docs/automated_testing.md

```diff
@@ -771,13 +771,19 @@ The output ALKiln creates includes:
 
 - A report of the result from all the tests.
 - Pictures of screens where ALKiln ran into errors or unexpected behavior.
+  - The HTML of pages where ALKiln ran into errors or unexpected behavior.
```

Comment:
Nit: This list item looks indented. Is that intentional? If it is intentional, maybe there should be a list of two items - the picture and the html?

Reply:
This was intentional; I treat the HTML and the pictures as kinda one thing everywhere else, (i.e. it's not a separate thing you have to think about, I want the idea of "screenshot" to include the picture and the HTML). I see how it's a bit confusing though. I'll clarify this a bit further when fixing up the typos in a different branch.